enhancements/etcd/automated-backups.md

```diff
@@ -223,6 +276,8 @@ This test would effectively run through the backup and restore procedure as foll
 - Step through the recovery procedure from the saved backup
 - Ensure that the control-plane has recovered and is stable
 - Validate that we have restored the cluster to the previous state so that it does not have the post-backup changes
+  - Ideally want a way to have all clients/operators restart their watches as they may be trying to watch revisions that do not exist post backup. Restarting the individual operators is one solution but it does not scale well for a large number of workloads on the cluster.
```

Comment:
> but it does not scale well for a large number of workloads on the cluster.

I think it would work, the problem is more: how do we know what is an operator that has a cached API object? 
In that sense, restarting all nodes would be better - but that indeed doesn't scale that well.